sycl/include/sycl/ext/intel/fpga_dsp_control.hpp

```diff
@@ -0,0 +1,79 @@
+//==------------ fpga_dsp_control.hpp --- SYCL FPGA DSP Control ------------==//
```

Comment:
PR description points to Gerrit review which is not publicly availble.
Could you point it to the commit/PR which is publicly available?

Reply:
I just checked the Clang change has not been merged to https://github.com/intel/llvm yet, so I'll just remove the Gerrit link there.